tests/operators/test_python.py

```diff
@@ -58,6 +64,17 @@
 ]
 
 
+@pytest.fixture()
+def pypi_server():
+    if not os.path.exists('/root/packages'):
+        os.makedirs('/root/packages')
+    process = subprocess.Popen(['pypi-server', '--port=8282'])
+    yield
+    process.kill()
+    shutil.rmtree('/root/packages')
+
+
+
```

Comment:
Where is this `pypi-serve` command coming from?

Simpler might be to just use `python -m http.server --directory ` as that is part of the standard library

```suggestion
@pytest.fixture()
def pypi_server(tmpdir):
    process = subprocess.Popen(['python', '-m', 'http.server', '--directory', str(tmpdir),  '--port=8282'])
    yield
    process.kill()


```

Reply:
I think this one should not changed, this due to the fact that we want to mimc the pypi server with all of its capabilities.
the pypi server is a package of course, installed for debugging (testing) only)